mono/tests/test-runner.cs

```diff
@@ -310,6 +316,110 @@ class TestInfo {
 			}
 		}
 
+		TimeSpan test_time = DateTime.UtcNow - test_start_time;
+		XmlWriterSettings xmlWriterSettings = new XmlWriterSettings ();
+		xmlWriterSettings.NewLineOnAttributes = true;
+		xmlWriterSettings.Indent = true;
+		using (XmlWriter writer = XmlWriter.Create ("TestResults_runtime.xml", xmlWriterSettings)) {
+			// <?xml version="1.0" encoding="utf-8" standalone="no"?>
+			writer.WriteStartDocument ();
+			// <!--This file represents the results of running a test suite-->
+			writer.WriteComment ("This file represents the results of running a test suite");
+			// <test-results name="/home/charlie/Dev/NUnit/nunit-2.5/work/src/bin/Debug/tests/mock-assembly.dll" total="21" errors="1" failures="1" not-run="7" inconclusive="1" ignored="4" skipped="0" invalid="3" date="2010-10-18" time="13:23:35">
+			writer.WriteStartElement ("test-results");
+			writer.WriteAttributeString ("name", "runtime-tests.dummy");
+			writer.WriteAttributeString ("total", (npassed + nfailed).ToString());
+			writer.WriteAttributeString ("failures", nfailed.ToString());
+			writer.WriteAttributeString ("not-run", "0");
+			writer.WriteAttributeString ("date", DateTime.Now.ToString ("yyyy-MM-dd"));
+			writer.WriteAttributeString ("time", DateTime.Now.ToString ("HH:mm:ss"));
+			//   <environment nunit-version="2.4.8.0" clr-version="4.0.30319.17020" os-version="Unix 3.13.0.45" platform="Unix" cwd="/home/directhex/Projects/mono/mcs/class/corlib" machine-name="marceline" user="directhex" user-domain="marceline" />
+			writer.WriteStartElement ("environment");
+			writer.WriteAttributeString ("nunit-version", "2.4.8.0" );
```

Comment:
Is this attribute required or just nice to have? I'd prefer just omitting it in the latter case instead of hardcoding the version. It's likely we'll forget to update this should we change nunit versions in the future...